client/src/components/RelatedItemsAndOutfit/Carousel.jsx

```diff
@@ -2,114 +2,153 @@ import React, { useCallback, useEffect, useState, createRef } from 'react';
 import Card from './Card.jsx';
```

Comment:
Since you aren't using hooks anymore, you can delete line 1 now.

Reply:
ahhh ty